### ISSUE TITLE
sequence_length(0) <= 15

### ISSUE THREAD

**Issue by aa642531** [Belval/CRNN#6]
hello,when i tried to train the model ,i meet the follows:
InvalidArgumentError (see above for traceback): sequence_length(0) <= 15.it happend at code ' loss = tf.nn.ctc_loss(targets, logits, seq_len) '  when feed datas

**Comment by aa642531** [comment]
@Belval

**Comment by Belval** [comment]
Please see https://github.com/Belval/CRNN/issues/5

**Comment by aa642531** [comment]
@Belval Hi,these days i found a way to solve this problem,but i am not sure whether it ' s right.
after conv and  map2seq,before feed into blstm,i add the code:
inputs=tf.reshape(inputs,[config.BATCH_SIZE,256,30])
I resize the all images to the fixed width 256,and it can run successfully.

**Comment by Belval** [comment]
Does it converge?

**Comment by Belval** [comment]
@aa642531 There was an error in the CNN part that wasn't returning the right dimension. Please retry with master.
